### PR TITLE
INFRA-749 Shut down and delete test H2 databases

### PR DESCRIPTION
**Pull request by rnicoll**
* Shut down on disk H2 databases to ensure they free resources after use (in memory automatically shuts down I believe)
* Delete H2 database from disk to free space

### CODE REVIEW

**Comment by lockathan** [comment]
Looks good, only comment is that we might need something similar in MockNetwork. I was thinking about whether this should go into 4.6 or 4.7, but it probably should go into 4.6.

**Comment by rnicoll** [comment]
@lockathan Done

**Review by lockathan — DISMISSED** [review]
LGTM

File: testing/node-driver/src/main/kotlin/net/corda/testing/node/DatabaseSnapshot.kt [hunk 1]
```diff
@@ -6,18 +6,19 @@ import java.nio.file.Path
 
 object DatabaseSnapshot {
     private const val previousCordaVersion: String = "4.5.1"
-    private const val databaseName: String = "persistence.mv.db"
+    const val DATABASE_FILENAME: String = "persistence.mv.db"
```
Comment (woggioni) [inline testing/node-driver/src/main/kotlin/net/corda/testing/node/DatabaseSnapshot.kt]: why did you remove the `private` access modifier? it doesn't look like you are using it outside of `DatabaseSnapshot` object

Comment (rnicoll) [reply testing/node-driver/src/main/kotlin/net/corda/testing/node/DatabaseSnapshot.kt]: Early edit that didn't get reverted, will in fact revert.

**Review by lockathan — APPROVED** [review]
LGTM
